docs/specification/2025-03-26/basic/authorization.mdx

```diff
@@ -63,248 +156,335 @@ audience. For instance:
    - For instance, an agent calls a secure MCP tool to check inventory at a specific
      store. No need to impersonate the end user.
 
-### 2.2 Example: authorization code grant
+MCP clients **MUST** support authorization code flow with PKCE, per
+OAuth 2.1 specification.
```

Comment:
Would this be better worded as:
```suggestion
MCP clients **MUST** implement PKCE when using the authorization code flow, per OAuth 2.1 specification."
```
since clients do not "support" flows.